package.json

```diff
@@ -1,5 +1,6 @@
 {
   "private": true,
+  "packageManager": "pnpm@7.15.0",
```

Comment:
Can we change this to something > 8? v7 is no longer maintained. https://pnpm.io/versions

FWIW, I am using `v8.2.0`.